lib/compilers/nvcc.ts

```diff
@@ -103,18 +95,12 @@ export class NvccCompiler extends BaseCompiler {
         return result;
     }
 
-    /**
-     *
-     * @param {*} result
-     * @param {string} outputFilename
-     * @param {import('../../types/features/filters.interfaces').ParseFiltersAndOutputOptions} filters
-     */
-    async postProcess(result, outputFilename, filters) {
+    override async postProcess(result, outputFilename: string, filters: ParseFiltersAndOutputOptions) {
         const maxSize = this.env.ceProps('max-asm-size', 64 * 1024 * 1024);
         const optPromise = result.hasOptOutput ? this.processOptOutput(result.optPath) : Promise.resolve('');
         const asmPromise = (
             filters.binary
-                ? this.objdump(outputFilename, {}, maxSize, filters.intel, filters.demangle, filters)
+                ? this.objdump(outputFilename, {}, maxSize, filters.intel, filters.demangle, false, false, filters)
```

Comment:
Typescript uncovered a bug here, `staticReloc` and `dynamicReloc` weren't being passed. Is passing false for both appropriate?

Reply:
I think so: @dkm I think is the one to know for certain.

Reply:
Yes, passing false is valid in case of doubt. Using true will add extra parameters to `objdumper`, which may or may not be a good idea (don't know anything about  `nvcc` tools)